[11.0.0-preview5+] Can't debug in VSCode on .NET 6

**Describe the bug:**
When using v11.0.0-preview5 onward, debugging a .NET 6 application with Visual Studio Code no longer allows you to inspect variables. It gives me the following error:
`error CS1705: Assembly 'AvaloniaTest' with identity 'AvaloniaTest, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' uses 'System.Private.CoreLib, Version=7.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' which has a higher version than referenced assembly 'System.Private.CoreLib" mit Identität "System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e'`

**To Reproduce:**

1. Clone this repo: https://github.com/psyGamer/AvaloniaTest
2. Create a breakpoint somewhere (for me it was the constructor of MainWindow)
3. Start the debugger in VSCode and hit the breakpoint
4. No variables are shown and typing `this` in the debug console throws the error

**Expected behavior**
Be able to inspect the variables like in v11.0.0-preview4 and before.

**Desktop:**

- OS: Arch Linux

**Aditional Context:**
Switching to .NET 7 solves this problem, however changing the .NET version just for debugging isn't really an option.

--- a/src/Avalonia.Build.Tasks/XamlCompilerTaskExecutor.cs
+++ b/src/Avalonia.Build.Tasks/XamlCompilerTaskExecutor.cs
@@ -149,7 +149,7 @@ namespace Avalonia.Build.Tasks
             {
                 var ctor = asm.MainModule.ImportReference(typeSystem.GetTypeReference(asmMetadata).Resolve()
                     .GetConstructors().First(c => c.Parameters.Count == 2).Resolve());
-                var strType = asm.MainModule.ImportReference(typeof(string));
+                var strType = asm.MainModule.TypeSystem.String;
                 var arg1 = new CustomAttributeArgument(strType, "AvaloniaUseCompiledBindingsByDefault");
                 var arg2 = new CustomAttributeArgument(strType, defaultCompileBindings.ToString());
                 asm.CustomAttributes.Add(new CustomAttribute(ctor) { ConstructorArguments = { arg1, arg2 } });

